crates/holochain_types/src/app/app_manifest/app_manifest_v1.rs

```diff
@@ -68,17 +75,17 @@ pub struct AppSlotDnaManifest {
     /// Note that since this is flattened,
     /// there is no actual "location" key in the manifest.
     #[serde(flatten)]
-    pub(super) location: Option<mr_bundle::Location>,
+    pub location: Option<mr_bundle::Location>,
 
     /// Optional default properties. May be overridden during installation.
-    pub(super) properties: Option<YamlProperties>,
+    pub properties: Option<YamlProperties>,
 
     /// Optional fixed UUID. May be overridden during installation.
```

Comment:
This UUID field, is it expected to be (and validated as) an actual [UUID as formally defined](https://en.wikipedia.org/wiki/Universally_unique_identifier), or can it be any string of bytes? May be good to be explicit about the expectations for this field.

Reply:
So far it's just an arbitrary string. I'll note this as an item to discuss.

Reply:
I think it'd be nice to have as an arbitrary string, for use cases in which people want to join a space by typing in a string. Higher risk of accidental collision vs UUIDs, I realise, but it's nice to be able to permit that use case.